packages/workspace/src/utils/name-utils.ts

```diff
@@ -23,7 +23,7 @@ export function names(
  * hypenated to UpperCamelCase
  */
 export function toClassName(str: string): string {
-  return toCapitalCase(toPropertyName(str));
+  return toCapitalCase(toPropertyName(str.replace(/[^a-zA-Z\d]/g, '')));
```

Comment:
Because Next.js files can contain `[]` characters, we need to strip these off for class name.

Reply:
could the `[` and `]` characters just get added to the `toPropertyName` regex replace?

Reply:
Hmm, yeah that would be good.